src/surveys/components/SingleResponse.tsx

```diff
@@ -0,0 +1,49 @@
+import { VNode, h } from 'preact';
+import { useState } from 'preact/hooks';
+import { Button, Header } from '../../components';
+import { ID, Question, Theme } from '../../types';
+
+interface SingleResponseProps {
+  question: Question;
+  onAnswered: (questionId: ID, answerId: ID) => void;
+  theme?: Theme;
+}
+
+export default function SingleResponse({
+  question,
+  onAnswered,
+  theme,
+}: SingleResponseProps): VNode {
+  const [selectedOption, setSelectedOption] = useState<ID>('');
+
+  const handleSubmit = (e: h.JSX.TargetedEvent<HTMLFormElement, Event>) => {
+    e.preventDefault();
+    if (!selectedOption) return;
+
+    onAnswered(question.id, selectedOption);
+  };
+
+  return (
+    <form className={'max-w-sm space-y-4'} onSubmit={handleSubmit}>
+      <Header title={question.label} color={theme?.question} />
```

Comment:
@AfeezGL There's an optional `description`, if provided. It should be a sub-header as shown in the image below.
<img width="384" alt="Screenshot 2023-08-20 at 09 06 06" src="https://github.com/Formily/formily-web/assets/113894296/005dc3df-072f-4005-bfcf-373662f82cb7">